src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -86,6 +102,59 @@ public void handlesSocket() throws IOException {
     }
 
     /**
+     * BkBasic can handle HTTP persistent connections.
+     * @throws Exception If some problem inside
+     * @checkstyle ExecutableStatementCountCheck (2 lines)
+     */
+    @Test
+    @SuppressWarnings("PMD.DoNotUseThreads")
+    public void handlesPersistentConnection() throws Exception {
+        final int port = 8080;
```

Comment:
@mstzn36 this is not acceptable. how can you be sure that port 8080 is available on the build machine?